vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -61,7 +58,6 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.4.0",
             "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.21",
             "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.14",
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v0.7.5", # for k8s 1.20.x
```

Comment:
Does this mean 2019-containerd doesn't support 1.20.x?

Reply:
Will fix